src/utils.js

```diff
@@ -463,37 +469,19 @@ export default class Utils {
             + `&i=${encodeURIComponent(md.content_id)}&u=${encodeURIComponent(md.user_id)}`
             + '&aqe=%'; //endpointUrl
 
-        let b = JSON.stringify(this.buildIngest(ur, ct, sp)); //body
+        let b = JSON.stringify(this.buildIngest(ur, ct, sp));
         let eb = null; //encodedPayload
 
+        if (dg && dg.outputLog) {
+            console.dir(JSON.parse(b));
```

Comment:
It's not necessary to call `JSON.parse()` if this part is put before `b` variable is stringified.